.travis.yml

```diff
@@ -1,6 +1,19 @@
 sudo: false
-install: ./autogen.sh
 language: c
 compiler:
     - clang
     - gcc
+os:
+    - linux
+    - osx
+matrix:
+  exclude:
+    - os: osx
+      compiler: gcc
+  fast_finish: true
+
+before_install:
+    - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew update; fi
+install: ./autogen.sh
+script: ./configure --prefix=${TRAVIS_BUILD_DIR}/install_dir &&
+         make install V=1 && ${TRAVIS_BUILD_DIR}/install_dir/bin/fi_info -v
```

Comment:
So, I'm not sure if make test should do anything other than run fi_info.  But it would be nice if travis ran fabtests.  Others can chime in if that level of testing should be part of make test or limited to this script.